xml/System.Net.WebSockets/ValueWebSocketReceiveResult.xml

```diff
@@ -49,10 +49,10 @@
         <Parameter Name="endOfMessage" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="count">To be added.</param>
-        <param name="messageType">To be added.</param>
-        <param name="endOfMessage">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="count">The number of bytes received.</param>
+        <param name="messageType">The type of message that was received.</param>
+        <param name="endOfMessage">Whether this is the final message.</param>
```

Comment:
Strictly speaking "endOfMessage" doesn't represent the "final message".

These receive result's represent "fragments" of a WebSocket message.  They are not strictly "frames" as defined by RFC 6455.  But represent pieces of a complete WebSocket message.  When the final fragment is returned back from this API, we set the "endOfMessage" to be true.  This corresponding to the WebSocket protocol receiving the FIN bit as described here: https://tools.ietf.org/html/rfc6455#section-5.2

So, instead of saying "Whether this is the final message".  It probably should say, "Indicates whether the message has been received completely."

Reply:
In fact, looking at this documentation, you should copy the words from here:
https://docs.microsoft.com/en-us/dotnet/api/system.net.websockets.websocketreceiveresult?view=netframework-4.8

This class, "ValueWebSocketResultResult" is really the same thing as the "WebSocketReceiveResult" class.  We added this new one with the "Value" prefix as an optimization for .NET Core.  But semantically, the things in the class are the same thing as "WebSocketReceiveResult".

Reply:
```suggestion
        <param name="endOfMessage">Indicates whether the message has been received completely.</param>
```

Reply:
Thanks. I'll copy all the words from the link you provided.